### DISCUSSION TITLE
Consider renaming `CameraAssembly` 's `camera_assembly_name` to `name`

### DISCUSSION THREAD

**Discussion by bruno-f-cruz** [AllenNeuralDynamics/aind-data-schema#856] [Ideas]
**Is your feature request related to a problem? Please describe.**
Most of the devices or even other classes always use `name`. This would make issues like #609 potentially much easier to solve in the future and also provide users with more consistency across classes.

**Describe the solution you'd like**
rename the field

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Comment by saskiad** [comment]
We use X_assembly_name consistently for assemblies (camera, ephys, laser, fiber). I think this was intentional, but I need to remember exactly why.

**Comment by galenlynch** [comment]
I +1 this, I think if everything had a `name` field it would be much simpler to write generic code. Different assemblies and devices have `name` (most devices), `assembly_name` (e.g. `DomeModule`), `camera_assembly_name`, `ephys_assembly_name`, to name just a few that I have encountered.

**Reply by dyf** [reply]
I prefer this as well, but it would be a significant change throughout the schema.

**Comment by dbirman** [comment]
Stale discussion, this was completed in March
